### ISSUE TITLE
Inaccurate timestamps

### ISSUE THREAD

**Issue by phanxuanquang** [sandrohanea/whisper.net#299]
Hello,

I found that the transcription has very bad timestamp in my most cases when I use it for English or Japanese. I tried both Base and Large-v3 models. How can I solve this issue?

This is the video I want to transcript: [Best wireless mic for filming on your phone | The Verge](https://youtu.be/3FkvBsFYLBo)

**Comment by phanxuanquang** [comment]
Hello @sandrohanea,

Thank you for your suggestions. However, I can't find the ways to set the RuntimeOptions.

Can you please give me some guidance or examples?

**Comment by sandrohanea** [comment]
Hello @phanxuanquang ,

In the released version you just need to call:

```csharp
RuntimeOptions.SetUseDtwTimeStamps(true);
```

Ensure you call it when your program starts before any WhisperFactory is created.

you'll need this using for the RuntimeOptions:
```
namespace Whisper.net.LibraryLoader;
```

In the newer version (not released yet, probably 1.7.4), it will be possible to configure when you create a WhisperFactory by providing a `WhisperFactoryOptions` object to the `WhisperFactory.FromBuffer()` or `WhisperFactory.FromPath()` methods.
